src/microbe_stage/MicrobeHUD.tscn

```diff
@@ -1955,6 +1957,17 @@ toggle_mode = true
 ActionIcon = ExtResource( 9 )
 ActionName = "g_toggle_engulf"
 
+[node name="EjectEngulfed" parent="ScrollContainer/HotBar" instance=ExtResource( 65 )]
```

Comment:
Did you fix the focus style here already? As engulfment is an actual toggle that is different from this. You should instead look at how the FireToxin button is made to copy a single press action button for this feature.

Reply:
Doesn't there also need to be this button but as a dummy button in the multicell hud as well to avoid crashing?

Reply:
I think it works as non-toggle button would. And about multicellular HUD, yes, I forgot.

Reply:
Looking at the overridden themes etc. properties in this text format, this looks much more like the toggle button than the toxin fire button. So I was wondering if something was wrong. Maybe you could check those property differences and make doubly sure that this is all like it should be.

Reply:
Well, I possibly copy-pasted toggle engulf button and then set it to work more like normal button, but I can inspect it and redo if needed

Reply:
I think redoing the button by copy pasting the toxin fire button would ensure there's nothing wrong with the new button.